adagrams/game.py

```diff
@@ -1,11 +1,174 @@
+import random
+
+LETTER_POOL_W_VALUES = {
```

Comment:
This is an interesting approach to this data structure! A nested dictionary can sometimes be tricky to work with, especially if it's going to be transformed or looped over (with or without nesting) so we'll see how it fares in the code below!